### PR TITLE
Error codes migration

### PR DESCRIPTION
**Pull request by mihail911**
Moved error codes module into HyperSearch dir.

Please review @oxtopus @scottpurdy

### CODE REVIEW

**Comment by scottpurdy** [comment]
remove data file...

File: nupic/support/nupic-default.xml [hunk 1]
```diff
@@ -113,7 +113,7 @@
 
 <property>
   <name>nupic.hypersearch.speculative.particles.default</name>
-  <value>1</value>
```
Comment (scottpurdy) [inline nupic/support/nupic-default.xml]: don't make this change

File: nupic/swarming/HyperSearch/__init__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2013, Numenta, Inc.  Unless you have an agreement
```
Comment (scottpurdy) [inline nupic/swarming/HyperSearch/__init__.py]: wrong year

File: nupic/swarming/HyperSearch/__init__.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
```
Comment (scottpurdy) [inline nupic/swarming/HyperSearch/__init__.py]: need single newline character at eof

File: nupic/swarming/HyperSearch/errorcodes.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2013, Numenta, Inc.  Unless you have an agreement
```
Comment (scottpurdy) [inline nupic/swarming/HyperSearch/errorcodes.py]: year

**Comment by scottpurdy** [comment]
done with review

**Comment by mihail911** [comment]
@scottpurdy

**Comment by scottpurdy** [comment]
We should avoid the duplication of ErrorCodes. Can we reomve it from nupic.support?

Otherwise this looks good.

**Comment by mihail911** [comment]
Migrating these changes to a clean PR

**Pull request closed without merging**